src/components/text-editor/prosemirror-adapter/plugins/trigger-plugin.ts

```diff
@@ -0,0 +1,352 @@
+// import { ListItem } from 'src/components/list/list-item.types';
+
+import { Node, NodeType, Schema } from 'prosemirror-model';
+import { Plugin, PluginKey, EditorState } from 'prosemirror-state';
+import { ReplaceStep, ReplaceAroundStep } from 'prosemirror-transform';
+import { EditorView } from 'prosemirror-view';
+import { CustomElement } from 'src/interface';
+
+export const triggerPluginKey = new PluginKey('triggerPlugin');
+
+export type TriggerCharacter =
```

Comment:
What did we say about this? Should we still keep it or do we just have to keep it for now?

Reply:
I can change it in a follow up PR, there are some more things to do than simply changing the type to string